CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 7.2.0
+  - Support multiple pipelines using the same database [#181](https://github.com/logstash-plugins/logstash-filter-geoip/pull/181)
+  - Add EULA doc
```

Comment:
@jsvd this version only works with Logstash 7.14+. I will modify the description. Do I need to do anything to make sure 7.13.x won't use this version?

Reply:
we can pin this requirement in the gemspec:

```ruby
  # and write a short sentence about why this is pinned, maybe with a link to an issue or comment
  s.add_runtime_dependency "logstash-core", ">= 7.14.0"
```